### ISSUE TITLE
Collection(Edm.String) support

### ISSUE THREAD

**Issue by AresEkb** [jaystack/jaydata#91]
Here is a metadata fragment:

``` xml
<Property Name="Sheets" Type="Collection(Edm.String)"/>
```

It seems that oDataProvider doesn't support collections (I'm using JayData 1.3 beta). I've changed a 741th line of oDataProvider.js from

``` javascript
'$data.Array': function (o) {
if (o === undefined) { return new $data.Array(); }
else if (o instanceof $data.Array) { return o; }
return JSON.parse(o); },
```

to

``` javascript
'$data.Array': function (o) {
if (o === undefined) { return new $data.Array(); }
else if (o instanceof $data.Array) { return o; }
else if (o.results instanceof Array) { return o.results; }
return JSON.parse(o); },
```

It works for me, but I think there should be a more general fix. Thanks!

**Comment by robesz** [comment]
Hello, 

Great, thanks for the info, we investigate the issue, but we would need to know how to reproduce this result on the server-side. Is it a WCF Data Service or WebAPI with OData extension? Could you please paste the code of your service method, which returns this result?
Thanks,
Robert

**Comment by AresEkb** [comment]
It's Web API with OData + Entity Framework 5.0.

Here is a model fragment:

``` c#
    public class File
    {
        public IEnumerable<string> Sheets
        {
            get { return new string[] { "one", "two", "three" }; }
        }
    }
```

Sheets is a calculated property.

Controller is derived from EntitySetController with a trivial overrides.

**Comment by robesz** [comment]
Thanks for the details, this issue is resolved by this commit: https://github.com/jaydata/jaydata/commit/b49b36c250ae7e053b0682808455f5959adcac49
So the code is available in the development branch and will be released with v1.3 next week after the QA.

**Comment by AresEkb** [comment]
Thanks!!!
